fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -498,17 +496,6 @@ public Void visitTimestampArithmeticExpr(TimestampArithmeticExpr node, Scope sco
                     funcOpName = String.format("%sS_%s", node.getTimeUnitIdent(), "add");
                 } else if (subDateFunctions.contains(node.getFuncName())) {
                     funcOpName = String.format("%sS_%s", node.getTimeUnitIdent(), "sub");
-                } else if (node.getFuncName().equals(FunctionSet.TIME_SLICE)) {
```

Comment:
why can delete this code

Reply:
TIME_SLICE not with TimestampArithmeticExpr.